src/index.js

```diff
@@ -20,7 +20,7 @@ const abortEvents = [
   'wheel',
   'DOMMouseScroll',
   'mousewheel',
-  'keyup',
+  'keydown',
```

Comment:
`keyup` was interfering with the click listener to start the animation if it was started using the enter key.